v20.04/Dockerfile.amd64

```diff
@@ -0,0 +1,26 @@
+FROM owncloud/base:20.04-amd64
+
+LABEL maintainer="ownCloud DevOps <devops@owncloud.com>" \
+  org.label-schema.name="ownCloud Server" \
+  org.label-schema.vendor="ownCloud GmbH" \
+  org.label-schema.schema-version="1.0"
+
+VOLUME ["/mnt/data"]
+EXPOSE 8080
+
+ENTRYPOINT ["/usr/bin/entrypoint"]
+CMD ["/usr/bin/owncloud", "server"]
+
+RUN apt-get update -y && \
+  apt-get upgrade -y && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
+
+ADD owncloud.tar.bz2 /var/www/
```

Comment:
Will we keep on calling owncloud.tar.bz2 to the artifact for 10.5 when it will include all the apps? Which of the new artifacts should we use for the docker file for 10.5?

Reply:
The owncloud.tar.bz2 mentioned in the Dockerfile is actually 'https://download.owncloud.org/community/testing/owncloud-complete-20200525-qa.tar.bz2'
.drone.star configures that. I think this is correct.

There is however already another owncloud tar in the base image. Afaik this is just there to test if the image is operational, and it is not exactly defined which version should be expected there. Ususally an an older owncloud tar. (The one where a new php-dependency was introduced, or similar...)